tests/unit/amo/pages/TestCollection.js

```diff
@@ -160,7 +156,10 @@ describe(__filename, () => {
   it('renders placeholder text if there are no add-ons', () => {
     const { store } = dispatchSignInActions();
 
-    _loadCurrentCollection({ store, addons: [] });
+    _loadCurrentCollection({
+      store,
+      addons: createFakeCollectionAddonsListResponse({ addons: [] }),
```

Comment:
I was already thinking about that before but it is pretty clear here: it is super confusing to have a prop called `addons`, which is in fact an API response object. I think `loadCurrentCollection()` should therefore receive a `response`, which would match its type (`CollectionAddonsListResponse`).

That would probably be less confusing because this patch is hard to understand. It works for sure, but... it is black magic 😅

Reply:
Yeah, I wasn't too happy about this either. Changing the name to be explicit about the fact that it's a response is definitely better. Thanks!